react-events-and-state/src/index.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom/client';
+
+export default function CustomButton(props) {
+  const [isClicked, setIsClicked] = useState(false);
+  function handleClick() {
+    setIsClicked(!isClicked);
+  }
+
+  if (isClicked) {
```

Comment:
Your code would be more succinct if you used `isClicked` to calculate the text. Then you wouldn't have to repeat all the JSX.